lib/commands.ts

```diff
@@ -1,38 +1,47 @@
-import { VisualRegressionTracker } from '@visual-regression-tracker/sdk-js'
+import { VisualRegressionTracker } from "@visual-regression-tracker/sdk-js";
 
 let vrt: VisualRegressionTracker;
-const { vrt_config } = Cypress.env()
+const { vrt_config } = Cypress.env();
 
-export const addTrackCommand = () => Cypress.Commands.add("track", {
-    prevSubject: ['optional', 'element', 'window', 'document'],
-}, (subject, name, options,) => {
-    let imagePath: string;
-    let pixelRatio: number;
-    const target = subject ? cy.wrap(subject) : cy;
-    return target.screenshot(name,
-        {
-            ...options,
-            onAfterScreenshot: (el, props) => {
-                imagePath = props.path
-                pixelRatio = props.pixelRatio
-                return options?.onAfterScreenshot
-            }
+export const addTrackCommand = () =>
+  Cypress.Commands.add(
```

Comment:
Codacy found an issue: ['Cypress' is not defined.](https://app.codacy.com/gh/Visual-Regression-Tracker/agent-cypress/pullRequest?prid=5918194)